lib/compiler/plugins-loader.ts

```diff
@@ -63,9 +64,10 @@ export class PluginsLoader {
       if (!plugin.before && !plugin.after && !plugin.afterDeclarations) {
         throw new Error(CLI_ERRORS.WRONG_PLUGIN(pluginNames[index]));
       }
-      const options = isObject(plugins[index])
-        ? (plugins[index] as PluginAndOptions).options || {}
-        : {};
+      const options =
+        plugins[index] !== null && typeof plugins[index] === 'object'
+          ? (plugins[index] as PluginAndOptions).options || {}
+          : {};
```

Comment:
```suggestion
      const options =
        plugins[index]?.options || {}
```

I guess?

Reply:
Yes, here too we can improve what I had done by writing a little less code. If it's okay for Kamil, I'll proceed with the change 👍